### ISSUE TITLE
UI: fix clear button on search input for chromium browsers

### ISSUE THREAD

**Issue by tametsi** [yabfc/yabfc#43]
Currently, our search component provides its own clear button. I.e. we got 2 on Chromium-based browsers. How to proceed?

- Remove Chromium clear button?
- Switch `type=input`? a11y?

**Comment by philxws692** [comment]
I would suggest that we remove the chromium clear button, since it does not fit into our design (atleast on my device)

<img width="119" height="81" alt="Image" src="https://github.com/user-attachments/assets/767c74c4-b33c-4f23-ac9d-1000f89cef84" />

Would that be any drawback for accessibility?

**Comment by tametsi** [comment]
Would favor this, too. I think the easiest solution would be to change input type from `search` to `text`. Effect on a11y should be ok as we have labeled everything properly.
